tests/casino_test.cpp

```diff
@@ -7,6 +7,19 @@ using bytes = std::vector<char>;
 static constexpr int64_t seconds_per_day = 24 * 3600;
 static constexpr int64_t seconds_per_month = 30 * seconds_per_day;
 
+static uint64_t get_token_pk(const std::string& token_name) {
```

Comment:
can we import get_token_pk from platform.hpp like we made it in blackjack?

Reply:
good idea but this requires adding `contracts` as a dependency first